config_defaults/subtests/docker_cli/run.ini

```diff
@@ -54,6 +54,20 @@ last_name_sticks = yes
 names_count = 1000
 exit_status = 0
 
+[docker_cli/run/run_tmpfs]
+cmd = 'sleep 5s'
```

Comment:
Suggestion for a bit of an overhaul: instead of this `sleep` and a separate docker command, why not remove the `--detach` from `run_options_csv` and use `cmd = findmnt ...` ? First, that simplifies the code enormously. Second, it's robust: with the sleep, you have a race condition because on a loaded system the `sleep` container may finish before the `findmnt` one starts. I hope this makes sense; please ask for clarification as needed.